config/locales/devise.en.yml

```diff
@@ -0,0 +1,62 @@
+# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
```

Comment:
Journey of Internationalization starts here...You may want to read more about it:
http://guides.rubyonrails.org/i18n.html